e2e_deployment_files/connectivity_deployment.template.json

```diff
@@ -731,7 +731,7 @@
                 "value": "<LogAnalyticsSharedKey>"
               },
               "LogAnalyticsLogType": {
-                "value": "testMetrics"
+                "value": "testMetricsV2"
```

Comment:
Again, why v2?

Reply:
Log types are like tables. I am changing the schema of the metric tags so I thought it best to create a new table. This also helped to make the downstream changes to the metrics dashboard / alerts, as they didn't immediately break with differently shaped data going into the existing table.